R/rdeephaven/R/table_ops.R

```diff
@@ -632,12 +644,16 @@ setGeneric(
 setMethod(
   "sort",
   signature = c(table_handle = "TableHandle"),
-  function(table_handle, by, descending = FALSE) {
+  function(table_handle, by, descending = FALSE, abs_col = FALSE) {
```

Comment:
can we use `abs`?

Reply:
Will create a conflict with base R's `abs()` in the scope of the function. Probably not a killer, but should be aware of it.

Reply:
The problem with `abs_col` is that it mixes with our existing nomenclature where `col` is a column name.  Here it is not.  `abs` makes it clear that it is how sorting would happen, as would something like `sort_abs`.